test/functional/feature_csv_activation.py

```diff
@@ -92,6 +94,7 @@ def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
         self.extra_args = [[
+            '-peertimeout=999999',  # bump because mocktime might cause a disconnect otherwise
```

Comment:
I take it this needs to be an even larger timeout than we currently [use in a few other places](https://github.com/bitcoin/bitcoin/blob/5446070418c1a3540d32c95831b9d1aefe8d6755/test/functional/feature_bip68_sequence.py#L46), i.e: `"-peertimeout=9999",  # bump because mocktime might cause a disconnect otherwise`? Otherwise would be good to settle on the same LARGE number everywhere.

Reply:
It only needs to be 1 second longer than the whole duration of the test. If the test takes 100 seconds, it needs to be 101. So `9999` seems enough for our purposes here.

Reply:
Maybe peertimeout should be set to ~infinity by default in our functional tests, unless opted out?